addons/hr_expense/static/src/views/expense_line_widget.js

```diff
@@ -15,7 +15,7 @@ export class ExpenseLinesListRenderer extends ListRenderer {
     /** @override **/
     async onCellClicked(record, column, ev) {
         const sheetId = this.env.model.root.resId;
-        const sheetThread = this.threadService.getThread('hr.expense.sheet', sheetId);
+        const sheetThread = this.env.services["mail.store"].Thread.get({ model: 'hr.expense.sheet', id: sheetId });
```

Comment:
I don't think this change is safe. If we don't have current knowledge of the thread, `sheetThread` is undefined.

`getThread` has the good property to getOrInsert a chatter thread.
I think it's worth keeping reliance on this function for the time being for chatter-related things.
Also I think `.get()` is a function that should be relied less and less. Reasoning is that `.get()` requires the code to have inserted record beforehand otherwise it crashes. `insert()` is much better.

We could move this function somewhere else, but IMO that's a minor issue and current implementation is fine.